File: backend/src/main/java/com/ufc/backend/backend/model/based/FeedbackBasedInterface.java
package com.ufc.backend.backend.model.based;

import com.ufc.backend.backend.model.feedback.Feedback;

import java.util.List;

public interface FeedbackBasedInterface {
void addFeedback(Feedback feedback);
List<Feedback> findAllFeedbacks();
void updateFeedback(Feedback feedback);
void deleteFeedback(Feedback feedback);
}

File: backend/src/main/java/com/ufc/backend/backend/model/based/FeedbackBasedInterface.kt
package com.ufc.backend.backend.model.based

import com.ufc.backend.backend.model.feedback.Feedback

interface FeedbackBasedInterface {
    fun addFeedback(feedback: Feedback)
    fun findAllFeedbacks(): MutableList<Feedback>
    fun updateFeedback(feedback: Feedback)
    fun deleteFeedback(feedback: Feedback)
}